cmd/virt-handler/virt_launcher.cil

```diff
@@ -53,8 +53,4 @@
     (allow process nfs_t (dir (mounton)))
     (allow process proc_t (dir (mounton)))
     (allow process proc_t (filesystem (mount unmount)))
-    ;
-    ; Allowing libvirtd to write to /sys files.
-    ; This permission is necessary when using SR-IOV, but is not needed for anything else.
-    (allow process sysfs_t (file (write)))
```

Comment:
I am not sure you can remove this as we still need to mount GPU devices at this stage. Maybe just change SR-IOV to GPU.

Reply:
maybe would worth to ask Jed or to try GPU out just in order to see if it affect it ?
git history can teach about it as well maybe, it was added after both 
SRIOV and GPU mounts were there and said SRIOV only.

Reply:
I think a simple common sense is enough here.
There is not difference really between the two devices in this context, both are passed into the VM in a similar manner (hostdev passthrough).

@alonSadan , you can send a separate PR to remove the GPU on top of this PR and remove this section as well.
It can then be reviewed by the relevant people.

Reply:
OK. removed. I will follow up with a PR, add the link here.

Reply:
Since there is no realy difference for those two devices in this context I think this should be addressed in this PR.
Why do we need to worry about it so much there is a comment the explicitly says its for SRIOV only?
If it was used for GPU I would expect for another comment.
Do we have a lane for GPU?

Reply:
code still mounts sysfs dir for GPU, code cant be removed without tested, either you test it or you dont touch it and doing it on a follow PR.

Reply:
You dont know GPU used it, you assume it is.
What will be the difference to remove it in another PR?

Reply:
and you dont know GPU doesnt need it, you assume it is.
see please all the discussion above, we dont remove code just because we assume something without testing it.

Reply:
draft PR for now: 
https://github.com/kubevirt/kubevirt/pull/4476

Reply:
I dont think we have a lane to test it

Reply:
> I dont think we have a lane to test it

I'll keep it draft then until it is verified